gradle-suppressible-error-prone/src/test/resources/suppression-test/AddNewSuppressionsAtEveryLevel.java

```diff
@@ -0,0 +1,53 @@
+// Args: -PerrorProneSuppress
```

Comment:
This will allow us to later test running `-PerrorProneApply -PerrorProneSuppress` together and check it does the right thing.